chalice/.chalice/config.json

```diff
@@ -1,12 +1,13 @@
 {
   "version": "2.0",
   "app_name": "dss",
-  "environment_variables": {},
+  "environment_variables": {
```

Comment:
This value only locates the file on the lambda's filesystem. It doesn't influence which credentials we give to the lambda.

Separately from this, we can (in make deploy/build_deploy_config.sh) parameterize which credentials we give to which stage (dev/staging/prod/custom) and to which lambda (dss-apiserver/dss-sync/dss-index).

(Separately, we have the option of switching to Lambda environment variables to transmit the contents of `gcp-credentials.json` - it takes up the majority of the 4KB env var size allowance, but it does fit (2KB gzipped+b64encoded). But then we'd have to figure out how to pass it to google cloud bindings, which expect a JSON file.)